Pak-Je-Kans.md

```diff
@@ -0,0 +1,47 @@
+# Wat wordt er gemaakt
+---
+We willen een plek maken op Mijn Amsterdam genaamd Pak Je Kans voorzieningencheck, hier kunnen mensen zien waar ze recht op hebben. Deze armoedevoorzieningen worden zo persoonlijk mogelijk gefilterd. Dit doen we met de gegevens die we van de burger hebben en krijgen, als de burger toegeeft dat we zijn of haar egevens gebruiken om deze voorzieningen te laten zien. Het doel van deze omgeving is ervoor zorgen dat de burger beter zicht heeft op wat ze allemaal recht hebben. De burgers kunnen het ook direct aanvragen.
+
+# Wat moet er gemaakt worden in de Back-End
+---
+* Er moeten files worden gemaakt met voorzieningen en voorwaarden erin.
+* Er moeten functies komen die deze voorwaarden toepassen op de voorzieningen en deze doorgeven aan de front end waar nodig is.
+* Ook moeten er tests worden geschreven om te checken of het allemaal werkt.
+
```

Comment:
Als we 3 bovenstaande puntjes hebben uitgevoerd, hebben we dan een werkende back-end?

Reply:
We hebben oa ook nog nodig:
- Netwerkconnectiviteit + bijbehorende env variabelen om connectie te maken met de bron api
- Achter TMA plaatsen
- Certificaten
- Jenkinsfile voor build en deploy
- Kibana en Sentry logging views op de NUC

Kun je dit verder nog bij @drakonen verifiëren?